homeassistant/helpers/device_registry.py

```diff
@@ -272,13 +272,14 @@ def __delitem__(self, key: str) -> None:
 
     def get_entry(
         self,
-        identifiers: set[tuple[str, str]],
         connections: set[tuple[str, str]] | None,
+        identifiers: set[tuple[str, str]] | None,
```

Comment:
let's revert this. There will still be custom components using positional arguments and this will mess up the registry big time.